crates/rspack_fs/src/async.rs

```diff
@@ -23,6 +23,8 @@ pub trait AsyncWritableFileSystem {
   /// Write a slice as the entire contents of a file.
   /// This function will create a file if it does not exist, and will entirely replace its contents if it does.
   fn write<P: AsRef<Path>, D: AsRef<[u8]>>(&self, file: P, data: D) -> BoxFuture<'_, Result<()>>;
+
+  fn rm<P: AsRef<Path>>(&self, dir: P) -> BoxFuture<'_, Result<()>>;
```

Comment:
I would in favor of the name provided by the original std impl. Could you change it to `remove_dir_all`?

Reply:
Thank you, it has been modified